stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/FootnoteDecorator.kt

```diff
@@ -106,8 +119,9 @@ internal class FootnoteDecorator(
         threadGuideline: View,
         anchorView: View,
         data: MessageListItem.MessageItem,
+        isThreadMode: Boolean,
     ) {
-        val isSimpleFootnoteMode = data.message.replyCount == 0 || data.message.isInThread()
+        val isSimpleFootnoteMode = data.message.replyCount == 0 || data.message.isInThread() || isThreadMode
```

Comment:
I think `data.message.isInThread()` is redundant here

Reply:
You're right. Thanks